stories/Articles/ArticleCard.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Meta } from "@storybook/react";
+import { ArticleCard } from "@/Components/Articles/ArticleCard";
+
+export default {
+    title: "Articles/ArticleCard",
+} as Meta<typeof ArticleCard>;
+
+export const Default = {
+    render: ({
+        collections,
+        article,
+    }: {
+        collections: Array<Pick<App.Data.Nfts.NftCollectionData, "image" | "slug">>;
+        article: any;
+    }) => {
+        return (
+            <ArticleCard
+                collections={collections}
+                article={article}
+            />
```

Comment:
add a few more variants, e.g.:
- 1 featured collection
- 3 featured collections
- 8 featured collections
- short title that is not truncated
- long title that is truncated
- add a grid overview where you show 4-5 cards next to each other to highlight responsiveness

Reply:
except for the 8 features collections that require a component rest is ready